#type error

# x=5
# y="hello"
# z=x+y
# print(z)#type error

#try catch lock to resolve it

# x=5
# y="hello"
# try:
#     z=x+y
# except TypeError:
#     print("Error: a int and str cannot e added")


x=5
y="hello"
try:
    z=x+y
except Exception as e:
    # print("Error: a int and str cannot e added")
    print(e)

    